apps/backend/src/routes/home/+page.svelte

```diff
@@ -0,0 +1,240 @@
+<!-- JavaScript code -->
+<script lang='ts'>
+  //Script to obtain current year
+  const currentYear = (new Date().getFullYear() as unknown) as string;
+
+  //Script to obtian Previous year
+  const previousYear = ((new Date().getFullYear() - 1) as unknown) as string;
+  
+</script>
+
+<!-- Landing Page - Card -->
+<div class="flex flex-row items-center justify-center h-screen">
+
+    <!-- Landing Page - Content -->
+    <div class=" w-full max-w-7xl h-screen p-10 rounded-md">
+        
+        <!-- Landing Page - Header -->
+        <div class="dropdown dropdown-bottom">
+            <div tabindex="0" role="button" class="btn m-1 text-2xl font-semibold normal-case px-6 rounded-full border-none bg-transparent">My Files
+                <span>
+                    <svg xmlns="http://www.w3.org/2000/svg" height="20" width="14" viewBox="0 0 320 512" fill="#fff">
+                        <!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M137.4 374.6c12.5 12.5 32.8 12.5 45.3 0l128-128c9.2-9.2 11.9-22.9 6.9-34.9s-16.6-19.8-29.6-19.8L32 192c-12.9 0-24.6 7.8-29.6 19.8s-2.2 25.7 6.9 34.9l128 128z"/>
+                    </svg>
+                </span>
+            </div>
+            <ul class="dropdown-content z-[1] menu p-2 shadow-2xl bg-base-100 rounded-box w-52 border">
+                <li>
+                    <a href="/">
+                        <svg xmlns="http://www.w3.org/2000/svg" height="16" width="12" viewBox="0 0 384 512" fill="#fff">
+                            <!--!Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2023 Fonticons, Inc.--><path d="M0 64C0 28.7 28.7 0 64 0H224V128c0 17.7 14.3 32 32 32H384V448c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V64zm384 64H256V0L384 128z"/>
+                        </svg>
```

Comment:
Save this svg into a file of it's own and use it dynamically via imports.

Reply:
Done the same as above, added a new file called fileicon.svg in static and used an `<img>` tag
![Uploading image.png…]()